pennylane/drawer/tape_mpl.py

```diff
@@ -203,21 +214,22 @@ def tape_mpl(tape, wire_order=None, show_all_wires=False, decimals=None, **kwarg
 
     PennyLane has inbuilt styles for controlling the appearance of the circuit drawings.
     All available styles can be determined by evaluating ``qml.drawer.available_styles()``.
-    Any available string can then be passed to ``qml.drawer.use_style``.
+    Any available string can then be passed via the kwarg ``style`` to change the settings for
+    that plot. This will not affect style settings for subsequent matplotlib plots.
 
     .. code-block:: python
 
-        qml.drawer.use_style('black_white')
-        fig, ax = tape_mpl(tape)
+        fig, ax = tape_mpl(tape, style='sketch')
 
-    .. figure:: ../../_static/tape_mpl/black_white_style.png
+    .. figure:: ../../_static/tape_mpl/sketch_style.png
             :align: center
             :width: 60%
             :target: javascript:void(0);
 
     You can also control the appearance with matplotlib's provided tools, see the
     `matplotlib docs <https://matplotlib.org/stable/tutorials/introductory/customizing.html>`_ .
-    For example, we can customize ``plt.rcParams``:
+    For example, we can customize ``plt.rcParams``. To use a customized appearance based on matplotlib's
+    ``plt.rcParams``, ``qml.draw_mpl`` must be run with style=None::
 
     .. code-block:: python
```

Comment:
This block is rendering incorrectly.  Sphinx can be rather non-intuitive and finicky, but I'm going to hypothesize that the problem is the `::` double colon on the line above. We can give it a try and find out.

Reply:
Thanks, that seems to have fixed it!